### PR TITLE
Support changes in preview pipeline template

### PR DESCRIPTION
**Pull request by hallipr**
- remove npm scripts in favor of common powershell scripts eng/scripts/(initialize|build|test).ps1
- remove common chsred code script in favor of direct invocations where appropriate
- remove duplicate typescript generation code

### CODE REVIEW

File: eng/scripts/test.ps1 [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#Requires -Version 7.0
```
Comment (m-nash) [inline eng/scripts/test.ps1:1]: The name of this file made me think this was some kind of temporary test script.  Can we call it run-tests or something like this so it sounds more purposeful?

File: eng/pipelines/preview.yml [hunk 2]
```diff
@@ -24,11 +32,11 @@ pool:
 extends:
   template: eng/pipelines/templates/stages/archetype-autorest-preview.yml@azure-sdk-tools
   parameters:
-    RegenerationJobCount: 20
+    RegenerationJobCount: 10
```
Comment (m-nash) [inline eng/pipelines/preview.yml:35]: What motivated this change?

**Review by m-nash — APPROVED** [review]

File: eng/pipelines/preview.yml (hunk 2 above)
Comment (hallipr) [reply eng/pipelines/preview.yml:35]: There was no considerable change in runtime when fanning out so wide.

Compare 20 legs:
image

to 10:
image

Comment (hallipr) [reply eng/pipelines/preview.yml:35]: Until we can use a weighted grouping, were stuck with some leg taking about 20 minutes.

File: eng/scripts/test.ps1 (hunk 1 above)
Comment (hallipr) [reply eng/scripts/test.ps1]: I renamed the scripts to:
- Initialize-Repository.ps1
- Build-Packages.ps1
- Test-Packages.ps1

**Pull request merged**